docs/admin/groups.md

```diff
@@ -0,0 +1,7 @@
+# User Groups
+
+Groups are helpful to define a set of common permissions for a group of users. It can be used with [RBAC](../rbac.md) to have a better access control.
```

Comment:
```suggestion
Groups are helpful for defining a set of common permissions for a group of users. It can be used with [RBAC](../rbac.md) to have a better access control.
```